qadence/backends/utils.py

```diff
@@ -98,10 +98,11 @@ def to_list_of_dicts(param_values: ParamDictType) -> list[ParamDictType]:
     if not param_values:
         return [param_values]
 
-    max_batch_size = max(p.size()[0] for p in param_values.values())
+    max_batch_size = max(p.size()[0] for p in param_values.values() if isinstance(p, Tensor))
```

Comment:
did you merge current main? i thought i added this recently

Reply:
well, the `main` is without this addition

Reply:
oh good catch, thanks!